src/main/java/de/unistuttgart/overworldbackend/data/PlayerTaskStatistic.java

```diff
@@ -0,0 +1,38 @@
+package de.unistuttgart.overworldbackend.data;
+
+import java.util.UUID;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.experimental.FieldDefaults;
+import org.hibernate.annotations.ColumnDefault;
+
+@Entity
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@FieldDefaults(level = AccessLevel.PRIVATE)
+public class PlayerTaskStatistic {
+
+  @Id
+  @GeneratedValue(generator = "uuid")
+  UUID id;
+
+  @ManyToOne
+  Playerstatistic playerstatistic;
+
+  @ManyToOne
+  MinigameTask minigameTask;
+
+  @ManyToOne
+  Lecture lecture;
+
+  long highscore = 0;
```

Comment:
```suggestion
  @Min(0)
  @Max(100)
  long highscore = 0;
```
Should we add min and max like in the DTO?